### DISCUSSION TITLE
404 페이지 gif

### DISCUSSION THREAD

**Discussion by NamgyungKim** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#270] [Ideas]
## 1. 귀여워서 채택(오른쪽에 글씨로 404Notfound라고 쓸예정)
![104152-404-super-sad](https://user-images.githubusercontent.com/87519250/174754504-96622d75-76ba-4623-9ce1-e9b08b9b3b75.gif)

## 2. 무난해서 채택(하단에 글씨로 404Notfound라고 쓸예정)
![98247-404-error](https://user-images.githubusercontent.com/87519250/174754509-f668faea-a25a-498a-a044-28ab259cdbe4.gif)

## 3. 우리 로고가 자동차라서 채택(오른쪽에 글씨로 404라고 쓸예정)
![70383-page-not-found-error-404](https://user-images.githubusercontent.com/87519250/174754519-27323454-6d9d-4b5e-8bad-58713507fbb4.gif)

그외... 좋은 아이디어 있으시면 남겨주세요

- [ ] 1.귀여워서 
- [ ] 2.무난해서 
- [x] 3.우리 로고가 자동차라서

**Comment by Hyevvy** [comment]
3번 .. 넘 귀여워요 ㅠ

**Comment by hyunjin0910** [comment]
전 3번 ㅎㅎ 메인 페이지 노랑이랑 잘어울려요

**Comment by YuHyun-P** [comment]
저도 3번!!!

**Reply by YuHyun-P** [reply]
1번 너무 귀여운데 프로필 이미지로 사용해도 될까요ㅋㅋㅋ
